From 5663aca2b258657db31611ac356325880d9177a3 Mon Sep 17 00:00:00 2001
From: flintman <flintman@flintmancomputers.com>
Date: Wed, 26 Nov 2014 14:52:04 -0500
Subject: [PATCH] build:  remove active checkjni be default

---
 core/main.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index f816b43..5735292 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -357,8 +357,6 @@ ifneq (,$(user_variant))
   ADDITIONAL_DEFAULT_PROPERTIES += ro.allow.mock.location=0
 
 else # !user_variant
-  # Turn on checkjni for non-user builds.
-  ADDITIONAL_BUILD_PROPERTIES += ro.kernel.android.checkjni=1
   # Set device insecure for non-user builds.
   ADDITIONAL_DEFAULT_PROPERTIES += ro.secure=0
   # Allow mock locations by default for non user builds

